konig-transform-beam/src/main/java/io/konig/transform/beam/BeamExpressionTransformImpl.java

```diff
@@ -447,7 +447,7 @@ public BlockInfo beginBlock(JBlock block) {
 	private IJExpression systime() {
 		AbstractJClass longClass = model.ref(Long.class);
 		AbstractJClass dateClass = model.ref(Date.class);
-		return longClass._new().arg(dateClass._new().invoke("getTime"));
+		return longClass._new().arg(dateClass._new().invoke("getTime").div(1000L));
```

Comment:
This change will return the number of seconds since the beginning of the epoch.

I think we need to be returning  microseconds.  
If that's true, then we should be using...
```
return longClass._new().arg(dateClass._new().invoke("getTime").mul(1000L);
```

Reply:
As we discussed in our last scrum meeting, the Beam interface to BigQuery is expected dateTime values expressed in seconds (not milliseconds or microseconds).  Uvashri is going to post a message to StackOverflow to find out why the API requires the time in seconds when the documentation states that BigQuery supports microseconds.

In the meantime, we'll accept the code as written (capturing the value in seconds).